cookbook/form/form_customization.rst

```diff
@@ -114,6 +114,12 @@ fragment needed to render every part of a form:
   but the CSS classes applied are the ones used to display the forms horizontally
   (i.e. the label and the widget in the same row).
 
+.. caution::
```

Comment:
yeah, as it's something to be aware of. That's a bit more than a note. (it's on the edge of note/caution)